README.md

```diff
@@ -143,23 +143,52 @@ docker run -it --rm \
   -w $(realpath "${PWD}") \
   --network host \
   vagrantlibvirt/vagrant-libvirt:latest \
-    vagrant status
+  vagrant status
 ```
 
-It's possible to define an alias in `~/.bashrc`, for example:
+It's possible to define a function in `~/.bashrc`, for example:
 ```bash
-alias vagrant='
+vagrant(){
   docker run -it --rm \
-    -e LIBVIRT_DEFAULT_URI \
-    -v /var/run/libvirt/:/var/run/libvirt/ \
-    -v ~/.vagrant.d:/.vagrant.d \
-    -v $(realpath "${PWD}"):${PWD} \
-    -w $(realpath "${PWD}") \
-    --network host \
-    vagrantlibvirt/vagrant-libvirt:latest \
-    vagrant'
+  -e LIBVIRT_DEFAULT_URI \
+  -v /var/run/libvirt/:/var/run/libvirt/ \
+  -v ~/.vagrant.d:/.vagrant.d \
+  -v $(realpath "${PWD}"):${PWD} \
+  -w $(realpath "${PWD}") \
+  --network host \
+  vagrantlibvirt/vagrant-libvirt:latest \
+  vagrant $@
+}
+
+```
+
+---
+To run with Podman you need to add
+
+```bash
+  --entrypoint /bin/bash \
```

Comment:
This skips https://github.com/vagrant-libvirt/vagrant-libvirt/blob/master/entrypoint.sh which does a number of set up steps that are normally required for things to work as expected e.g. have the internal user be in the correct group to access the libvirt socket. So you'll need to explain how this works as expected subsequently.

Reply:
I realized you need to run podman as root as well since a rootless container won't work. I think this is just temporary workaround until you can add official support/update `entrypoint.sh` but so far I've had no problem using this setup.

Reply:
The only effect I've encountered from this is any new files/folders created by Vagrant will be owned by root, any existing Vagrantfiles won't be affected which shouldn't break anything.

Reply:
Have you tried the 'edge' version of the image without overriding the entrypoint? The goal of the entrypoint is to ensure that when vagrant is executed it is with the correct UID/GID so that the files remain readable/writeable by the user.

Reply:
I just tried the `edge` version which seems to work so I removed the `--entrypoint /bin/bash` argument.